auditbeat/module/auditd/audit_linux.go

```diff
@@ -531,7 +589,7 @@ func (s *stream) ReassemblyComplete(msgs []*auparse.AuditMessage) {
 }
 
 func (s *stream) EventsLost(count int) {
-	lostMetric.Inc()
+	lostMetric.Add(int64(count))
```

Comment:
I don't completely understand the logic behind the original "lost" metric, but I thought this to be a mistake.

wdyt @andrewkroh ?

Reply:
At one point I changed the meaning behind the metric to be the number of times sequence number gaps were detected (`lostMetric.Inc()`). 

I did this because it seemed like value of `count` might not be accurate in all cases. IIRC I was seeing some wildly large `count` values.

Let's keep your change and modify the name of this metric to be `reassembler_lost`.

Reply:
I observed that audit events from the kernel always have a sequence number of zero. The only messages with a sequence number are the replies to our commands. That can cause the wrong count to be reported, yes.

I agree with the name change.

Reply:
Darn, I was hoping that would explain the crazy values I saw and that all we needed was to special case the zero seq in the `libaudit.Reassembler`.